docs/supported-services/beanstalk.rst

```diff
@@ -134,6 +134,12 @@ The Routing element is defined by the following schema:
     routing:
       type: <http|https>
       https_certificate # Required if you select https as the routing type
+      dns_names:
+       - <string> # Optional
+
+DNS Names
```

Comment:
The DNS Names header looks just about the same size as its enclosing Routing header when rendered. What would you say to removing the DNS Names header? To me it looks more clearly like it belongs in the Routing section and not on its own.